### DISCUSSION TITLE
Error with creating Arrow Files

### DISCUSSION THREAD

**Discussion by jshi7** [GreenleafLab/ArchR#1525] [Questions / Documentation]
Hello, 
I'm trying to create an ArrowFile and I keep getting this error:
 
<simpleError in validObject(r): invalid class “dgTMatrix” object: lengths of slots i and j must match>
I followed all the default tutorial parameters and only used one thread. I am using 10x fragments data. I run it on Sherlock, and the example dataset seems to work, so I'm not sure what the issue is. 

[ArchR-createArrows-74357945f688-Date-2022-07-27_Time-16-27-19.log](https://github.com/GreenleafLab/ArchR/files/9204323/ArchR-createArrows-74357945f688-Date-2022-07-27_Time-16-27-19.log)

I saw this issue thread but already have v. 1.0.2 https://github.com/GreenleafLab/ArchR/issues/1145

**Comment by rcorces** [comment]
please post errors to the Issues section and follow the provided template.
